static/flow/views/landing-page-data-set-view.js

```diff
@@ -271,7 +271,7 @@ var LandingPageDataSetView = function(options) {
             }
             //return;
             */
-            stopRecording();
+            stopRecording(base.show);
```

Comment:
I thought I made a comment on this in the refactoring PR but looks like it got lost...should this be `piSelectorPanel.stopRecording(base.show)`?

Reply:
It's scope allows it to be callable here.  This is one of those cases where many functions were made global in the original design.  I followed that style, but now I'm trying to pull it back and reduce scope on things.  I think this falls under my general tech debt request to analyze and tighten the scope of functions across the app.

Reply:
Weird! Alright, all looks fine then :)

Reply:
Yeah, I agree.  Sometime when you have a free hour or two, I would love to sit down and try to make a plan to rework some of the scope in Dataflow.  I know it isn't pressing now, but there is all sorts of odd scope that I would like to sort out.  I think many functions and variables should have a more limited scope.